content/millennium/r4/medications/medication_request.md

```diff
@@ -0,0 +1,144 @@
+---
+title: MedicationRequest | R4 API
+---
+
+# MedicationRequest
+
+* TOC
+{:toc}
+
+## Overview
+
+The Medication Request resource provides orders for all medications along with administration instructions for a
```

Comment:
I would prefer to not insert manual line breaks

Reply:
https://github.com/cerner/fhir.cerner.com/pull/332/commits/f8adf8960cba9c644ecdc5477491918f0f44486d#diff-91cabf44fbcf0c5aaa861d340e57d246L12